### ISSUE TITLE
Unhandled error event

### ISSUE THREAD

**Issue by aitorllj93** [ttezel/twit#178]
Hi! I've got this error with the latest version of the library (1.1.20) on Node v12
```javascript
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: socket hang up
    at createHangUpError (_http_client.js:215:15)
    at TLSSocket.socketCloseListener (_http_client.js:247:23)
    at TLSSocket.emit (events.js:129:20)
    at TCP.close (net.js:476:12)
```

**Comment by aitorllj93** [comment]
Here:
```javascript
this._stream = this._tw.stream('statuses/filter', this._track);
```
It enters in 'connect' event but crashes before 'connected'

**Comment by ttezel** [comment]
I think this should be fixed in twit v2.0. Closing this, but feel free to reopen if you get this again!
